lib/src/shared/test/java/com/couchbase/lite/BaseTest.java

```diff
@@ -151,14 +151,18 @@ protected void deleteDatabase(String dbName) throws CouchbaseLiteException {
         if ((dbDir == null) || !Database.exists(dbName, dbDir)) { return; }
 
         // sometimes, db is still in used, wait for a while. Maximum 3 sec
```

Comment:
is this comment still valid? 100 * 5 = 0.5 secs

Reply:
Good catch.  WIll fix.